scripts/setup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+sudo apt-get update -y
```

Comment:
Should probably do an upgrade too and I don't believe update requires -y, just upgrade:

Reply:
You're right, done in 27a1642d84c3319d67756924bb44afbf099c2f03